### ISSUE TITLE
Summary's rating bar won't turn yellow after the given rating

### ISSUE THREAD

**Issue by ioanarusiczki** [mozilla/addons-frontend#6265]
STR:
1. Load AMO dev and go to an add-ons/theme detail page which has no ratings given yet
2. Click a rating star (such as 5/5) and observe the review summary displayed on the right top corner

Expected result:
Number of ratings increased with one and the yellow bar changes color.

Actual result:
The yellow bar that indicated the 5 star rating is still greyed out.

Notes:
This issue is reproducible on AMO dev with FF62(Win10&Android 8.0).
Refreshing page will show the correct display.

![the yellow bar](https://user-images.githubusercontent.com/33448286/45624030-c43a9d00-ba91-11e8-9862-15f656922379.gif)

**Comment by kumar303** [comment]
Thanks for the report. This will be fixed in https://github.com/mozilla/addons-frontend/issues/5986

**Comment by kumar303** [comment]
This may be unrelated to #5986. I'm looking into it now.

**Comment by kumar303** [comment]
Good catch. I see what the problem is. There was a similar bug where the review count above the star graph was not getting updated. The same patch will fix that, too.

**Comment by ioanarusiczki** [comment]
@kumar303 I cannot reproduce this scenario anymore. It's verified fixed on AMO dev with FF62(Win10 and Android 8.0)

![fixed issue](https://user-images.githubusercontent.com/33448286/45950015-a11a6b00-c006-11e8-89e9-e07c122cc92b.gif)

**Comment by kumar303** [comment]
I had to revert this because of regressions. See https://github.com/mozilla/addons-frontend/pull/6399

I'm going to fix the original bug in a different way.

**Comment by ioanarusiczki** [comment]
I re-checked this issue today and it's not reproducible - AMO dev FF62 - Win10 &Android 8.0

![yelow bar is yellow now](https://user-images.githubusercontent.com/33448286/46289039-72653d00-c590-11e8-84ff-cbd03d512535.gif)
